horovod/mxnet/__init__.py

```diff
@@ -87,3 +93,6 @@ def broadcast_parameters(params, root_rank=0):
     for _, p in params:
```

Comment:
Since we're changing this piece, could we make `hvd.broadcast_parameters()` support a tuple, so that https://github.com/uber/horovod/blob/master/examples/mxnet_mnist.py#L120 could be simplified as advertised in `README.md` to:
```
hvd.broadcast_parameters(model.get_params(), root_rank=0)
```
?

Reply:
I prefer not to change it. I am working on supporting MXNet Gluon trainer API, where we have a different type of params to broadcast and it is not a tuple.

Reply:
OK, sounds good.  Can we fix README.md, for now, to use the correct procedure for the time being then?

Reply:
Updated